### DISCUSSION TITLE
TiKz usual commands menu, TiKz usual options, ....

### DISCUSSION THREAD

**Discussion by rafa4mat** [WinEdt-Team/WinEdt-Support#73] [1.3 Ideas and features requests]
Tikz usual commands (paths, rectangles, circles, ellipses, arc, node,... , arrows,.. to,... controls...), options,..., blend left,.., in=..., out=,.. pos=,... |-, -stealth,

**Comment by Alex-Simonic** [comment]
I don't think that another bloated menu would be particularly useful for this purpose. In any case, it would not be me creating (or using) a list of Tikz commands and options. If you want to give it a try you can append an extra TiKz menu by using the `[MAIN_MENU*]` section in your private settings and see if it is actually of any use. Other TiKz users should help you with this if there is any interest in such functionality (either as a menu, a guided auto-completion, or a custom-written AI-powered wizard). I am not familiar with TiKz and cannot help with this.
